controller/inviteController.js

```diff
@@ -0,0 +1,121 @@
+const Invite = require('../model/inviteModel');
+const HttpStatus = require('http-status-codes');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+const create = (req, res) => {
+
+    const user = req.decoded;
+
+    //Only admins can create invite objects
+    if (!user || !ValidationUtil.isUserAdmin(user.accessType)) {
+        return res.status(HttpStatus.FORBIDDEN).json({
+            status: StatusEnum['ERROR'],
+            message: 'Forbidden.'
+        })
+    }
+
+    const invite = new Invite(req.body);
+
+    invite.save().then(() => {
+        return res.status(HttpStatus.CREATED).json(invite);
+    }).catch(err => {
+        const errorMessage = ValidationUtil.buildErrorMessage(err, 'create', 'invite');
+        return res.status(HttpStatus.BAD_REQUEST).json({
+            status: 'Error',
+            message: errorMessage
+        });
+    });
+};
+
+const getById = (req, res) => {
+
+    const inviteId = req.params.id;
+    const user = req.decoded;
+
+    Invite.findById(inviteId).then(invite => {
+        if(invite) {            
+            return res.status(HttpStatus.OK).json(invite);
+        }
+        else{ 
+            return res.status(HttpStatus.NOT_FOUND).json();
+        }
+    }).catch(err => {
+        return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
+            status: StatusEnum['ERROR'],
+            message: 'Internal server error.'
+        });
+    })
+
+};
+
+const remove = (req, res) => {
+    
+    const user = req.decoded;
+
+    //Only admins can remove invite objects
+    if (!user || !ValidationUtil.isUserAdmin(user.accessType)) {
+        return res.status(HttpStatus.UNAUTHORIZED).json({
+            status: StatusEnum['ERROR'],
+            message: 'Forbidden.'
+        })
+    }
+   
+    const inviteId = req.params.id;
+    
+    Invite.findOneAndDelete({_id: new ObjectId(inviteId)}).then(invite => {
+           if(invite) {
+               return res.status(HttpStatus.OK).json(true);
```

Comment:
Should this be the official convention? If delete is successful, return `true`, else `false`. Because, I think that currently all of us are sending the deleted object. This is your area of expertise, so enlighten us what the best practice is. 

I will tag @zirlman and @srkacha so that we know how to proceed forwards.

Reply:
I returned an object but now when I think about it, returning a Boolean makes more sense because deleted objects should be of no use to us.

Reply:
We SHOULD return 204 No content if delete was successful.